Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/Diagnostics/AnalyzerReleases.Shipped.md

```diff
@@ -1,6 +1,12 @@
 ; Shipped analyzer releases
 ; https://github.com/dotnet/roslyn-analyzers/blob/master/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
 
+## Release 1.7.3
```

Comment:
i manually updated this file with the new expected minor version number - is this the correct way to do it?

Reply:
Since it is a new minor version the patch number will reset back to `0`. So the version will be `1.7.0`.